apps/cookbook/src/app/showcase/slides-showcase/slides-showcase.component.html

```diff
@@ -1,39 +1,84 @@
-<div class="example">
-  <p>
-    <em>
-      <strong>Please note:</strong>
-      The slides do not like beeing switched between chrome-devtools mobileview and standard
-      webview. Please refresh your browser after changing from web to mobile and visa versa
-    </em>
-  </p>
-  <cookbook-example-viewer [html]="slidesExample.template" [ts]="slidesExample.codeSnippet">
-    <cookbook-slides-example #slidesExample></cookbook-slides-example>
-  </cookbook-example-viewer>
-
-  <h4>Slides properties:</h4>
-  <cookbook-api-description-properties
-    [properties]="properties"
-  ></cookbook-api-description-properties>
-  <br />
-  <h4>Slides Events:</h4>
-  <cookbook-api-description-events [events]="events"></cookbook-api-description-events>
-  <p>
-    <em>
-      <strong>Note:</strong>
-      Loading lots of data in many cards can result in an unwanted pause while the slides content
-      are beeing rendered. (The data is recieved, spinner dissapears but seconds passes before the
-      cards are rendered) The notations:
-      <br />
-      <code>
-        *kirbySlide="let slide; let i = index"
-        <br />
-        [slideIndex]="i"
-        <br />
-        [activeSlideIndex]="selectedSlideIndex
-        <br />
-      </code>
-      gives you means to pass info to each slide´s content about what index it has and compare that
-      to the activeSlideIndex, and thereby a strategy to render only a few cards in advance.
-    </em>
-  </p>
-</div>
+<p>
+  Slides are a collection of cards you can navigate horizontally in, either swipe or click. Slides
+  are great to contain content without using a lot of space, but the content has to be closely
+  related.
+</p>
+
+<p>
+  <em>
+    <strong>
+      *Please note: It is possible to set
+      <code>[simpleSlider]="true"</code>
+      if you want to revert back to the old
+      <code>kirby-slides</code>
+      design
+    </strong>
+  </em>
```

Comment:
Consider semantics and punctuation (`<em>` is used for [spoken emphasis and not to add importance](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#em_vs._strong))
```suggestion
  <strong>
    *Please note:</strong> It is possible to set
    <code>[simpleSlider]="true"</code>
    if you want to revert back to the old
    <code>kirby-slides</code>
    design.
```